definition/data/sheets/CaseEventToComplexTypes.json

```diff
@@ -5,5 +5,23 @@
     {"ID": "CaseDocument","CaseEventID": "ReviewedPaperResponse","CaseFieldID": "caseDocuments","ListElementCode": "receivedDatetime","EventElementLabel": "Document submitted on","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "MANDATORY"},
     {"ID": "ScannedDocument","CaseEventID": "ReviewedPaperResponse","CaseFieldID": "scannedDocuments","ListElementCode": "url","EventElementLabel": "Document Link","EventHintText": "","FieldDisplayOrder": 1,"DisplayContext": "OPTIONAL"},
     {"ID": "ScannedDocument","CaseEventID": "ReviewedPaperResponse","CaseFieldID": "scannedDocuments","ListElementCode": "type","EventElementLabel": "Document Type","EventHintText": "","FieldDisplayOrder": 2,"DisplayContext": "MANDATORY"},
-    {"ID": "ScannedDocument","CaseEventID": "ReviewedPaperResponse","CaseFieldID": "scannedDocuments","ListElementCode": "subtype","EventElementLabel": "Document Subtype","EventHintText": "","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"}
-]
\ No newline at end of file
+    {"ID": "ScannedDocument","CaseEventID": "ReviewedPaperResponse","CaseFieldID": "scannedDocuments","ListElementCode": "subtype","EventElementLabel": "Document Subtype","EventHintText": "","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"},
+    
+    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "extraOrderDirection","CaseFieldID": "otherDirections","EventElementLabel": "Select direction","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "OPTIONAL"},
+    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "extraDocUploadList","CaseFieldID": "otherDirections","EventElementLabel": "Extra statements for document upload","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "MANDATORY"},
+    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "expertReports","CaseFieldID": "otherDirections","EventElementLabel": "Extra statements for expert report","EventHintText": "","FieldDisplayOrder": 1,"DisplayContext": "OPTIONAL"},
+    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "otherDirectionHeaders","CaseFieldID": "otherDirections","EventElementLabel": "Direction header","EventHintText": "e.g. Upload contracts /Confirm documents are original","FieldDisplayOrder": 2,"DisplayContext": "MANDATORY"},
```

Comment:
/Confirm in caps?

Reply:
Actually I wanted to mention the field labels 'Upload' and 'Confirm', made look better